src/Aurora.Scrapers/ResultData/FootfetishBooruResultData.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Aurora.Scrapers.ResultData;
+
+public class FootfetishBooruResultData : SearchResultData
+{
+    public FootfetishBooruResultData(string[] tags)
+    {
+        Tags = tags;
+    }
+
+    public string[] Tags { get; set; }
```

Comment:
why array

Reply:
We want to use this class as a value-object, so it would not make sense to have it be IEnumerable. Alternatively, we do not plan on adding anything to it, so it would not make sense to have it be a List.